assets/environment-interpreter/activate/activate.d/attach-command.bash

```diff
@@ -4,19 +4,6 @@
 #   a. defeat the shell's normal startup scripts
 #   b. source the relevant activation script
 #   c. invoke the command in one of "stdin" or "-c" modes
-if [ -n "$FLOX_TURBO" ]; then
-  # "turbo command" mode: simply invoke the provided command and args
-  # from *this shell* without paying the cost of invoking the userShell.
-  if [ -n "${FLOX_SET_ARG0:-}" ]; then
-    # Wrapped binary from `flox build`.
-    exec -a "$FLOX_SET_ARG0" "$@"
```

Comment:
Were we using this at all? It looks like we may only be using it in the build wrapper at this point

Reply:
It exists expressly for the build wrapper case, yes. Recall that the wrapper was made for only that one case and was later repurposed for builds.

Reply:
I may need to take that back .. when running `flox activate -- command` it's important that it finishes with `exec -a` to set the entry in the process table to `command`. If we've missed or dropped that along the way then that's a problem. We should probably add a test for that at some point.

Reply:
> when running flox activate -- command it's important that it finishes with exec -a

That's a separate code path, but I think we're using exec

> It exists expressly for the build wrapper case, yes
As this script isn't used for the build wrapper, I believe this specific branch is dead code (there's a separate `FLOX_SET_ARG0` code path in the build wrapper)